modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.scss

```diff
@@ -35,6 +35,7 @@
 }
 
 @include responsiveCards(3840px, 40rem);
```

Comment:
This line creates a range between 2160px and 3840px, which includes 2560px, so your change in the next line is not necessary.